core/src/main/java/com/oath/oak/NovaManager.java

```diff
@@ -49,6 +49,7 @@ public long allocated() {
     @Override
     public Slice allocateSlice(int size, Allocate allocate) {
         Slice s = allocator.allocateSlice(size, allocate);
+        s.setVersion(globalNovaNumber.get());
```

Comment:
was this a bug? any test covers this case now?

Reply:
No, it wasn't a bug. Version was not part of the Slice, I have added it.